go/vt/mysqlctl/xtrabackupengine.go

```diff
@@ -156,6 +156,9 @@ func closeFile(wc io.WriteCloser, fileName string, logger logutil.Logger, finalE
 // and an overall error.
 func (be *XtrabackupEngine) ExecuteBackup(ctx context.Context, params BackupParams, bh backupstorage.BackupHandle) (complete bool, finalErr error) {
 
+	if params.IncrementalFromPos != "" {
```

Comment:
I understand that this doesn't make sense for xtrabackup. Is there a way for us (in a separate PR) to support xtrabackup's own incremental backups?

Reply:
Possibly, yes. I haven't looked into it at all.